doc/source/train/user-guides/persistent-storage.rst

```diff
@@ -1,62 +1,387 @@
-Configuration and Persistent Storage
-====================================
-
 .. _persistent-storage-guide:
 
-.. _train-run-config:
+.. _train-log-dir:
+
+Configuring Persistent Storage
+==============================
+
+A Ray Train run produces a history of :ref:`reported metrics <train-monitoring-and-logging>`,
+:ref:`checkpoints <train-checkpointing>`, and :ref:`other artifacts <train-artifacts>`.
+You can configure these to be saved to a persistent storage location.
+
+.. figure:: ../images/persistent_storage_checkpoint.png
+    :align: center
+    :width: 600px
+
+    An example of multiple workers spread across multiple nodes uploading checkpoints to persistent storage.
+
+By default, these training outputs are saved in a local directory under ``~/ray_results``.
+This is sufficient for single-node setups or distributed training without saving
+model checkpoints or artifacts, but some form of external persistent storage such as
```

Comment:
> distributed training without saving model checkpoints or artifacts

This seems to be impractical, let's delete this? 

Also, should we not mention `~/ray_results` folder here, and introduce it in the local storage session? Users can freely decide where the outputs goes in local storage, and `~/ray_results` is a default value.